test/parallel/test-module-wrapper.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+require('../common');
+const fixtures = require('../common/fixtures');
+const { execFileSync } = require('child_process');
+
+const cjsModuleWrapTest = fixtures.path('cjs-module-wrapper.js');
+const node = process.argv[0];
```

Comment:
```suggestion
const node = process.execPath;
```

(To match most other test files.)